thermochem/janaf.py

```diff
@@ -0,0 +1,231 @@
+# janaf.py
+
+# This module gets thermodynamic datat from the JANAF database.
+# Files are downloaded from the NIST servers as needed and then cached locally.
+#
+# Zack Gainsforth
+#
+# Funding by NASA
```

Comment:
I'm going to convert this to a module docstring so it shows up in the documentation rather than a big block comment